doc/source/reference/code_support.rst

```diff
@@ -30,6 +30,8 @@ each supported output format using yt.
 +-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
 | Castro                |     Y      |  Y [#f3]_ |   Partial  |   Y   |    Y     |    Y     |     N      |   Full      |
 +-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| CfRadial              |    N/A     |    N/A    |     N/A    |   Y   |    Y     |    Y     |     Y      |   [#f5]_    |
```

Comment:
So this is strange, but I think that the way it's written these are correct, but for the intent of the table the first should probably be "Yes", N/A, and I *think* "Yes"?  Do we provide any metadata?

Reply:
> Do we provide any metadata?

Is that what would normally end up in `ds.parameters`? This frontend does implement a `_parse_parameter_file` and initializes an empty dict for `ds.parameters` ([link to src](https://github.com/zssherman/yt/blob/793a273c33f0ad85b5aa7aaae7144e3c8a21617f/yt/frontends/cf_radial/data_structures.py#L267)), but that method just sets all the domain-related attributes and doesn't actually put anything into `parameters`. So does that mean no metadata?

Reply:
Yeah, let's leave this as-is.

Reply:
changes I just pushed have Y, N/A, Y now. I do think the metadata handling could be improved here, but not in this PR. could copy over metadata into `parameters` or could expose the underlying xarray handle for exploration?

Reply:
Sounds good.  Future!